tools/run_tests/run_tests_matrix.py

```diff
@@ -252,7 +261,7 @@ def _create_portability_test_jobs(extra_args=[], inner_jobs=_DEFAULT_INNER_JOBS)
                               configs=['dbg'],
                               platforms=['linux'],
                               arch='default',
-                              compiler='python3.4',
+                              compiler='python3.5',
```

Comment:
This issue should also affect Basic Tests Linux since that also uses `tools/run_tests/helper_scripts/build_python.sh`